### ISSUE TITLE
Incorrect documentation for vision-language models with language modeling head

### ISSUE THREAD

**Issue by Aparna-B** [huggingface/hub-docs#1789]
**Doc request**
For vision-language models like GitForCausalLM, the documentation for returned value `logits` say:

"Prediction scores of the language modeling head (scores for each vocabulary token before SoftMax)."

However, the logits returned also include predictions for image tokens (line [here](https://github.com/huggingface/transformers/blob/f4fc42216cd56ab6b68270bf80d811614d8d59e4/src/transformers/models/git/modeling_git.py#L1459); comment in code is also incorrect).
 
Request: Correct documentation for the `logits` returned

**Comment by anuragbhaskar0720** [comment]
This is my first attempt at an issue, how do I assign myself the task?

**Comment by Wauplin** [comment]
@Aparna-B where exactly have you spotted this issue? And would you like to open a PR to fix it? (or @anuragbhaskar0720 if you are up for it?) :)
